docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/sparse/fill-empty-rows-16.rst

```diff
@@ -0,0 +1,107 @@
+FillEmptyRows
+======================
+
+
+.. meta::
+  :description: Learn about FillEmptyRows-16 - a sparse operation, which
+                can be performed on four required input tensors.
+
+**Versioned name**: *FillEmptyRows-16*
+
+**Category**: *Sparse*
+
+**Short description**: Fills empty rows of an input sparse tensor with a default value.
+
+**Detailed description**:
+
+Operation FillEmptyRows is an implementation of ``tf.sparse.fill_empty_rows``.
+
+The input sparse tensor is represented by the three inputs: 
+
+* ``indices``
+* ``values``
+* ``dense_shape``
+
+For each row in the input sparse tensor, this operator checks if the row is empty. If the row is empty, the operator adds an entry with the specified default value at index `[row, 0, ..., 0]`. The input may have empty columns at the end, which will not be affected by this operation.
+
+The output sparse tensor will be in row-major order and will have the same shape as the input, but with updated `output_indices` and `output_values`.
+
+This operator also returns a boolean vector indicating which rows were filled with the default value: ``empty_row_indicator[i] = True`` if row ``i`` was an empty row.
+
+**Attributes**: FillEmptyRows-16 operation has no attributes.
+
+**Inputs**:
+
+* **1**: ``default_value`` a scalar of type *T* to be inserted into the empty rows. **Required.**
+* **2**: ``values`` 1D tensor containing the values of type *T* to be inserted at the specified indices. **Required.**
+* **3**: ``dense_shape`` 1D tensor of type *T_IDX* indicating the shape of the dense tensor. **Required.**
+* **4**: ``indices`` 2D tensor of type *T_IDX* indicating the positions at which ``values`` are placed in the sparse tensor. **Required.**
+    It is of shape ``[M, N]``, where:
+
+    * ``M`` is the same as the length of the ``values`` input.
+    * ``N`` is equal to the rank of ``dense_shape``.
+
+**Outputs**:
+
+* **1**: ``output_indices`` 2D tensor of type *T_IDX* indicating the positions at which ``output_values`` are placed in the sparse tensor.
+    It is of shape ``[M', N]``, where:
```

Comment:
do we really need generalization of tf.raw_ops.SparseFillEmptyRows? May be to consider only 2D sparse tensor that is only needed by TF?
I think ND case can be converged to 2D case by reshape

Reply:
If only 2D is needed, let's limit it. Applied.